src/api/spec/controllers/about_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe AboutController, type: :controller do
+  render_views # NOTE: This is required otherwise Suse::Validator.validate will fail
+
+  describe '#index' do
+    before do
+      get :index, params: { format: :xml }
+    end
+
+    it { expect(response.status).to eq(200) }
```

Comment:
Minor thing, but this could also be ``` expect(response).to have_http_status(200) ```

One could even replace the 200 with :success